js/repl/Repl.tsx

```diff
@@ -102,6 +102,31 @@ function hasOwnProperty(obj, string) {
   return Object.prototype.hasOwnProperty.call(obj, string);
 }
 
+function provideDefaultOptionsForExternalPlugins(pluginName, babelVersion) {
+  switch (pluginName) {
+    case "@babel/plugin-proposal-decorators": {
+      if (compareVersions(babelVersion, "7.21.0") >= 0) {
+        return { version: "2023-01" };
```

Comment:
```suggestion
      if (compareVersions(babelVersion, "7.22.0") >= 0) {
        return { version: "2023-05" };
      } else if (compareVersions(babelVersion, "7.21.0") >= 0) {
        return { version: "2023-01" };
```

This will be released soon